CDE_schema.rnc

```diff
@@ -73,10 +102,12 @@ image =
     & element source { text }?
     & authors?
   }
-history =
-  element event {
+status_attrs = 
     attribute date { xsd:date },
     attribute status {
       "proposed" | "approved" | "published" | "deprecated"
     }
+history =
```

Comment:
I'm not sure that we need to nest stautus inside of two other tags

Reply:
"Status" isn't nested in multiple tags... it's directly on the "version" element (see the sample XML for examples).

Reply:
Understood, sorry for the confusion.